endpoint.go

```diff
@@ -723,14 +723,6 @@ func (ep *endpoint) sbLeave(sb *sandbox, force bool, options ...EndpointOption)
 }
 
 func (n *network) validateForceDelete(locator string) error {
-	if n.Scope() == datastore.LocalScope {
-		return nil
-	}
-
-	if locator == "" {
-		return fmt.Errorf("invalid endpoint locator identifier")
-	}
-
```

Comment:
Since the locator check is pretty much a noop its ok to remove it. But lets remove the function completely. No point in having a function that does nothing.